src/adapter.js

```diff
@@ -70,14 +113,17 @@ class VueAdapter extends Adapter {
 	updateVueComponent(view) {
 		const component = this._source.find(view.handle);
 
+		// TODO: Figure out how to pass path. view.handle?
```

Comment:
I wasn't able to test this part. How would I trigger this?

Reply:
With changes in a nested component - but looks good